README.md

```diff
@@ -43,6 +43,7 @@ A curated list of awesome **open-source** data visualizations frameworks, librar
 - [Echarts](https://github.com/ecomfe/echarts) - Highly customizable and interactive charts ready for big datasets.
 - [Epoch](https://github.com/epochjs/epoch) - Perfect to create real-time charts.
 - [Google Charts](https://developers.google.com/chart) - Interactive charts for browsers and mobile devices.
+- [GraphicsJS](http://www.graphicsjs.org) - Lightweight JS graphics library with intuitive API, based on SVG/VML.
```

Comment:
This library got removed since their site does not use HTTPS, can we use GitHub instead?

```suggestion
- [GraphicsJS](https://github.com/anychart/graphicsjs) - Lightweight JS graphics library with intuitive API, based on SVG/VML.
```

Reply:
This is likely the right change so I'll commit it, if you disagree, please open a new PR. I'm trying to work on this repo once a week.